packages/eslint-plugin/docs/rules/ban-ts-comment.md

```diff
@@ -1,4 +1,4 @@
-# Bans `// @ts-<directive>` comments from being used (`ban-ts-comment`)
+# Bans `// @ts-<directive>` comments from being used or requires descriptions after directive (`ban-ts-comment`)
 
 TypeScript provides several directive comments that can be used to alter how it processes files.
 Using these to suppress TypeScript Compiler Errors reduces the effectiveness of TypeScript overall.
```

Comment:
I can't comment on the line... but you'll need to update the docs to show the new option and briefly explain what it does

Reply:
sorry I overlooked this: added in https://github.com/typescript-eslint/typescript-eslint/pull/2099/commits/fda8ee1ceaf272889322d416785632840d90b6ee